packages/api/src/derives/staking/index.ts

```diff
@@ -1,6 +1,5 @@
 export { queryStakingAccountInfo as accountInfo } from './stakingAccount';
 export * from './overview';
-export * from './query';
```

Comment:
doesn't build dues to type issues, it isn't used from what I can see so opting to remove

Reply:
added this so that we can remove decorating api with staking derived queries in apps/UI project..

Reply:
ah so will this break the UI?

Reply:
no.. it won't... the UI is decorating it atm.